Sprint-1/implement/dedupe.js

```diff
@@ -1 +1,6 @@
-function dedupe() {}
+function dedupe(arr) {
+    return [...new Set(arr)]
```

Comment:
Can you explain how ` [...new Set(arr)]` creates an array without any duplicates in `arr`?